src/test/java/com/google/sps/CommentAnalysisTest.java

```diff
@@ -44,25 +44,19 @@
 /** This is a JUnit test for sentiment mockedAnalysis */
 @RunWith(JUnit4.class)
 public class CommentAnalysisTest {
-  private static final float testScore = 0.23f;
+  private static final float TEST_SCORE = 0.23f;
   private static final ArrayList<Double> SCORE_IN_RANGE =
       new ArrayList<>(Arrays.asList(0.001, 0.002, 0.003, 0.005, -0.1, -0.2));
   private static final ArrayList<Double> EDGE_SCORE =
       new ArrayList<>(Arrays.asList(1.0, -1.0, 0.0));
   private static final ArrayList<Double> SYMMETRIC_SCORE =
       new ArrayList<>(Arrays.asList(0.5, 0.9, -0.5, -0.9));
-  private static final ArrayList<Double> ALL_OUTSIDE_SCORE =
-      new ArrayList<>(Arrays.asList(-2.0, -3.0, 3.0, -100.2));
   private static final ArrayList<Double> ONE_OUSIDE_SCORE =
       new ArrayList<>(Arrays.asList(-2.0, -1.0, 0.0));
```

Comment:
Move them into their own tests as well, just like what you did in testExceptionAllOutsiderScore.